src/styles/index.scss

```diff
@@ -1,3 +1,86 @@
+$stopwatch-size: 80vmin;
+$center-size: 5vmin;
+$minutes-hand-hight: 20vmin + $center-size / 2;
+$minutes-hand-width: 3vmin;
+$seconds-hand-hight: 32vmin + $center-size / 2;
```

Comment:
There is a typo in the variable name `$seconds-hand-hight`. It should be `$seconds-hand-height` to correctly spell 'height'.